### ISSUE TITLE
Should Admin Sets be shown from the show works view based on Flipflop.assign_admin_set?

### ISSUE THREAD

**Issue by jkim-ru** [samvera/hyrax#2162]
### Descriptive summary

Hyrax 1.0.4: We're wondering if Admin Sets should be shown from the show works view based on ``Flipflop.assign_admin_set?``.

### Rationale

In our Hyrax-based application, we'd like to hide references to Admin Sets until we decide to use it. When we turned off admin sets from the "Administration" > "Settings" section of the application, we still saw the default admin set being shown in the show view of our work.

### Expected behavior

We expected the reference to admin sets to also be hidden from the works show view when we turned off the ability to assign admin sets.

### Actual behavior

We still see it in the works show view.

### Related work

Here's our application-specific ticket: https://github.com/dibbs-vdc/ccql/issues/27

We made a code modification in ``app/views/hyrax/base/_relationships_parent_rows.html.erb`` here: https://github.com/dibbs-vdc/ccql/commit/29db7f1000d3880b547c749f60e9dc54fc25fcf8#diff-dd2c2760bf99fd4d253ac3d1095fb2c4R29

Basically, we changed:

```
<%= presenter.attribute_to_html(:admin_set, render_as: :faceted) %>
```

to 

```
<% if Flipflop.assign_admin_set? %>
  <%= presenter.attribute_to_html(:admin_set, render_as: :faceted) %>
<% end %>
```

This might be related to existing issue: https://github.com/samvera/hyrax/issues/2050

**Comment by jcoyne** [comment]
This sounds reasonable.  In the future, if you have a question like this, please ask on the tech call or Slack or the mailing list.  Let's keep github issues only for bugs/features.

**Comment by jkim-ru** [comment]
My apologies.  I'll stick to slack or the GG forum.

**Comment by jkim-ru** [comment]
duplicate of https://github.com/samvera/hyrax/issues/2258
